### ISSUE TITLE
Aplite emulator doesn't find JS with config

### ISSUE THREAD

**Issue by ygalanter** [pebble/cloudpebble#196]
A JS file exists that correctly shows config page on Basalt. On Aplite attempt to show config page results in message:

*"[PHONE] pebble-app.js:?: No JS found, can't show configuration.*

**Comment by cjwilliams** [comment]
@ygalanter, perhaps a silly question, but was the app running at the time?

**Comment by ygalanter** [comment]
Yup, compiled and ran on emulator, clicked gear icon to open options
pop-up,  clicked "App Config". On Basalt page opens, Aplite shows that
message.
